### DISCUSSION TITLE
Hybrid Infrastructure IP management

### DISCUSSION THREAD

**Discussion by midhunkalaivasan** [Azure/ipam#276] [General]
I am planning to test this Azure IPAM 3.0. Just having a query, is it possible to manage hybrid Infra using this Azure IPAM tool ?
Any import IP/Subnet options available so that we can able to import the onpremise subnet details from excel or csv and manage it from here?

**Comment by DCMattyG** [comment]
Hi @midhunkalaivasan, I think it's important to understand that the Azure IPAM tool is **primarily** a tool for managing IP addresses in the Azure context. That said, the next version will expand the concept of "External Networks" and be able to manage Networks/Subnets/Endpoints that are completely outside of Azure.

This next version is due out soon and doesn't have import/export to CSV just yet, but it's definitely something I can look at adding.

Do you have a specific format the CSV would be in, or a tool you'd be generating the CSV from?
